metadata.json

```
@@ -23,7 +23,7 @@
     },
     {
       "name": "puppetlabs-firewall",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "version_requirement": ">= 1.0.0 <= 2.5.0"
```

We always allow the whole major release, because minor releases should not introduce breaking changes. Please change it to:
```suggestion
      "version_requirement": ">= 1.0.0 < 3.0.0"
```